R/get_recall_man.R

```diff
@@ -1,32 +1,31 @@
 #ua <- user_agent("http://github.com/WrathSmith/caRecall")
 #ua
-#' get_recall_numr
+#' get_recall_man
 #'
-#' @param recall_number An integer
+#' @param manufacturer A string
 #'
-#' @return text output
+#' @return dataframe
 #' @export
 #'
 #' @examples
 #' \dontrun{
-#' get_recall_num(1977044)
+#' get_recall_manu('Nissan')
 #' }
-get_recall_num <- function(recall_number) {
+get_recall_manu <- function(manufacturer) {
```

Comment:
This wasn't the code I had committed.  I was getting some errors from git and had thought I resolved them apparently I had not.